### PR TITLE
added navbar

### PR DESCRIPTION
**Pull request by eertl2**
Added a centered navbar to the top of the page. Links only connect to home page.

snip

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,6 +15,13 @@
 	<body>
 		<div id="header">
 			<img id="header-logo" src="t-logo.png"/>
+			<div id="navbar">
```
Comment (Avery2) [inline index.html]: instead of `id="navbar"` you can use `nav`
```suggestion
			<nav>
```

File: index.html [hunk 2]
```diff
@@ -15,6 +15,13 @@
...
+				<ul class="nav">	
+					<li><a href="./index.html">What is FOSS?</a></li>
+					<li><a href="./index.html">About Us</a></li>
+					<li><a href="./index.html">Our Leadership</a></li>
+				</ul>
+			</div>
```
Comment (Avery2) [inline index.html]:
```suggestion
			</nav>
```

File: index.css [hunk 3]
```diff
@@ -64,6 +64,21 @@ h2 {
 	background-color: #ffffff;
 }
 
+#navbar li {
```
Comment (Avery2) [inline index.css]:
```suggestion
nav li {
```

File: index.css [hunk 4]
```diff
@@ -64,6 +64,21 @@ h2 {
...
+	list-style-type: none;
+	float: left;
+}
+
+#navbar ul{
```
Comment (Avery2) [inline index.css]:
```suggestion
nav ul {
```

**Comment by Avery2** [comment]
Also, see if you can make the font the same size as the body, and maybe add some spacing

**Comment by eertl2** [comment]
added nav

**Pull request merged**